src/applications/disability-benefits/4142/config/form.js

```diff
@@ -0,0 +1,252 @@
+              'ui:widget': 'radio',
+              'ui:options': {
+                labels: {
+                  yes: 'Yes',
+                  no: 'No, please get them from my doctor.',
+                },
+              },
+            },
+            'view:privateRecordsChoiceHelp': {
+              'ui:description': medicalRecDescription,
+            },
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              'view:uploadRecs': {
+                type: 'string',
+                'enum': ['Yes', 'No, please get them from my doctor.'],
+              },
+              'view:privateRecordsChoiceHelp': {
+                type: 'object',
+                properties: {},
+              },
+            },
+          },
+        },
+      },
+    },
+    chapterServiceHistory: {
+      title: 'Supporting Evidence',
+      pages: {
+        [formPages.treatmentHistory]: {
+          path: 'treatment-history',
+          title: 'Supporting Evidence',
+          uiSchema: {
+            'ui:description': letUsKnow,
+            [formFields.privateMedicalProvider]: {
+              'ui:options': {
+                itemName: 'Provider',
+                viewField: PrivateProviderTreatmentView,
+                hideTitle: true,
+              },
+              items: {
+                dateRange: dateRangeUI(
+                  'Approximate date of first treatment',
+                  'Approximate date of last treatment',
+                  'End of treatment must be after start of treatment',
+                ),
+                privateProviderName: {
+                  'ui:title': 'Name of private provider or hospital',
+                },
+                privateProviderCountry: {
+                  'ui:title': 'Country',
+                },
+                privateProviderStreetAddress: {
+                  'ui:title': 'Street address',
+                },
+                privateProviderState: {
+                  'ui:title': 'State',
+                },
+                privateProviderCity: {
+                  'ui:title': 'City',
+                },
+                privateProviderPostalCode: {
+                  'ui:title': 'Postal code',
+                  'ui:options': {
+                    widgetClassNames: 'usa-input-medium',
+                  },
+                },
+                privatePrimaryPhoneNumber: {
+                  'ui:title': 'Primary phone number',
+                  'ui:widget': PhoneNumberWidget,
+                  'ui:options': {
+                    widgetClassNames: 'va-input-medium-large',
+                  },
+                  'ui:errorMessages': {
+                    pattern: 'Phone numbers must be 10 digits (dashes allowed)',
+                  },
+                },
+                limitedConsent: {
+                  'ui:title':
+                    'I give consent, or permission, to my doctor to release only records related to [condition].',
```

Comment:
Ahh, yes, the dreaded "How do I use `formData` in {thing}?" conundrum. I don't actually remember the specifics off the top of my head, but I think it'll amount to something like
```js
'ui:title': (formData) => 'Words go here ${formData.path.to.stuff}'
```

I basically _always_ have to `console.log(formData)` in that callback to see what `formData` actually _is_ in this case. Sometimes it's the entire form, sometimes it's a specific chunk of data. Just depends on where the callback is being used.

:thinking: Or, indeed, if `'ui:title'` even accepts a callback... :grimacing:

Like I said, I don't remember off the top of my head.

Reply:
The trouble here is that `condition` is probably not going to be a part of `formData`, because this form doesn't have any place for a veteran to indicate them. I'm not sure if there's a way around this.

Reply:
I will take note of this for when we fully integrate of 526.